app/helpers/rooms_helper.rb

```diff
@@ -10,4 +10,12 @@ def format_duration(duration_in_seconds)
       'Unavailable'
     end
   end
+
+  def midi?(content_type)
+    if ['audio/midi', 'audio/x-midi'].include?(content_type)
+      return true
+    else
+      return false
+    end
```

Comment:
that's pretty terrible 😈 `['audio/midi', 'audio/x-midi'].include?(content_type)` will do

Reply:
what have i done 😞